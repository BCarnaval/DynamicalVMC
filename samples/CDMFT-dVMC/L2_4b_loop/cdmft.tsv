model	omega	E_kin	E_pot	U	eb1_1	eb2_1	eb3_1	eb4_1	mu	t	tb1_1	tb2_1	tb3_1	tb4_1	ave_mu	ave_t	ave_U_1	ave_eb1_1	ave_eb2_1	ave_eb3_1	ave_eb4_1	ave_mu_1	ave_t_1	ave_tb1_1	ave_tb2_1	ave_tb3_1	ave_tb4_1	E0_1	sector_1	n_1	githash	version	GS_consistency	iterations	dist_function	distance	diff_hybrid	omegaH	script	time	
2L_4b_gen	0.49241003	-1.0233592	-0.8066739	4	0.36486427	-0.18482331	0.35809954	-0.18455164	1	1	0.34964094	0.31757483	0.34892926	0.31665837	0.85243905	-1.0233592	nan	nan	nan	nan	nan	0.88046346	-0.6070889	nan	nan	nan	nan	-4.146225	R0:N6:S0:1	0.88046346	20a4d9d	v2.1.5	Y	17	sharp_wc_2.0_b_25	 4.04e-02	 2.46e-02	 0.49241003	general_bath_1D.py	2023-07-26@10:34
2L_4b_gen	0.5067386	-1.0176744	-0.24928001	4	0.38843159	-0.20778129	0.38803619	-0.20479559	1.1	1	0.33574129	0.32211288	0.33519561	0.3216382	0.86557481	-1.0176744	nan	nan	nan	nan	nan	0.93239996	-0.63920227	nan	nan	nan	nan	-4.3310387	R0:N6:S0:1	0.93239996	20a4d9d	v2.1.5	Y	25	sharp_wc_2.0_b_25	 3.83e-02	 2.73e-02	 0.50673860	general_bath_1D.py	2023-07-26@10:44
2L_4b_gen	0.53160284	-0.99190017	-1.4345724	4	0.42200838	-0.22040785	0.42316944	-0.21977556	1.2	1	0.32066098	0.32488834	0.32067008	0.32488684	0.9012085	-0.99190017	nan	nan	nan	nan	nan	0.92726734	-0.62065942	nan	nan	nan	nan	-4.5788368	R0:N6:S0:1	0.92726734	20a4d9d	v2.1.5	Y	25	sharp_wc_2.0_b_25	 3.69e-02	 9.37e-03	 0.53160284	general_bath_1D.py	2023-07-26@10:55
2L_4b_gen	0.80885436	-0.99295361	0.82321722	4	0.99936457	-0.36548398	0.99837624	-0.36543936	1.3	1	0.34593243	0.38036857	0.34586274	0.38031156	0.921195	-0.99295361	nan	nan	nan	nan	nan	0.97927979	-0.68476396	nan	nan	nan	nan	-5.3114605	R0:N6:S0:1	0.97927979	20a4d9d	v2.1.5	Y	27	sharp_wc_2.0_b_25	 1.83e-02	 1.28e-02	 0.80885436	general_bath_1D.py	2023-07-26@11:06
2L_4b_gen	0.9243826	-0.9651495	2.2795786	4	1.2022596	-0.42074029	1.202238	-0.42066955	1.4	1	0.36082102	0.39191672	0.36080344	0.39190493	0.99863068	-0.9651495	nan	nan	nan	nan	nan	0.99180759	-0.6713	nan	nan	nan	nan	-5.7209277	R0:N6:S0:1	0.99180759	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 1.55e-02	 8.42e-03	 0.92438260	general_bath_1D.py	2023-07-26@11:12
2L_4b_gen	1.2357438	-1.0038328	2.4632528	4	1.4024649	-0.57045804	1.4023986	-0.57048237	1.5	1	0.39503424	0.40881756	0.39502177	0.40881895	1.0017429	-1.0038328	nan	nan	nan	nan	nan	0.99934874	-0.70855143	nan	nan	nan	nan	-6.5557786	R0:N6:S0:1	0.99934874	20a4d9d	v2.1.5	Y	22	sharp_wc_2.0_b_25	 6.13e-03	 7.62e-03	 1.2357438	general_bath_1D.py	2023-07-26@11:21
2L_4b_gen	1.4940613	-0.95373685	1.2432628	4	1.4153547	-0.69470678	1.4154165	-0.69471688	1.6	1	0.41405303	0.41724334	0.41405913	0.41724688	0.99875773	-0.95373685	nan	nan	nan	nan	nan	0.9978022	-0.66313227	nan	nan	nan	nan	-7.2634948	R0:N6:S0:1	0.9978022	20a4d9d	v2.1.5	Y	29	sharp_wc_2.0_b_25	 1.67e-03	 3.47e-03	 1.4940613	general_bath_1D.py	2023-07-26@11:33
2L_4b_gen	1.6113935	-0.99454245	0.81443722	4	1.3507583	-0.75072219	1.3507922	-0.75076141	1.7	1	0.41666804	0.42290969	0.41667764	0.42291545	1.0042072	-0.99454245	nan	nan	nan	nan	nan	1.0034709	-0.69648947	nan	nan	nan	nan	-7.6860109	R0:N6:S0:1	1.0034709	20a4d9d	v2.1.5	Y	28	sharp_wc_2.0_b_25	 3.31e-03	 6.67e-03	 1.6113935	general_bath_1D.py	2023-07-26@11:44
2L_4b_gen	1.9858304	-0.96268674	1.2072996	4	1.3389666	-0.93690139	1.3389431	-0.93687972	1.8	1	0.42973817	0.42798028	0.42973078	0.42797698	0.99893745	-0.96268674	nan	nan	nan	nan	nan	0.99886365	-0.6719127	nan	nan	nan	nan	-8.653206	R0:N6:S0:1	0.99886365	20a4d9d	v2.1.5	Y	21	sharp_wc_2.0_b_25	 9.60e-04	 4.32e-03	 1.9858304	general_bath_1D.py	2023-07-26@11:53
2L_4b_gen	2.1953612	-0.98681329	0.70325072	4	1.2333686	-1.0389955	1.2333699	-1.0390605	1.9	1	0.43422612	0.43223492	0.43422644	0.4322422	0.9976387	-0.98681329	nan	nan	nan	nan	nan	0.99753702	-0.6897384	nan	nan	nan	nan	-9.2680011	R0:N6:S0:1	0.99753702	20a4d9d	v2.1.5	Y	21	sharp_wc_2.0_b_25	 1.16e-03	 7.67e-03	 2.1953612	general_bath_1D.py	2023-07-26@12:02
