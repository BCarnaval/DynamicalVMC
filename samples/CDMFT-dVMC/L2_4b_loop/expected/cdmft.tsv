model	omega	E_kin	E_pot	U	eb1_1	eb2_1	eb3_1	eb4_1	mu	t	tb1_1	tb2_1	tb3_1	tb4_1	ave_mu	ave_t	ave_U_1	ave_eb1_1	ave_eb2_1	ave_eb3_1	ave_eb4_1	ave_mu_1	ave_t_1	ave_tb1_1	ave_tb2_1	ave_tb3_1	ave_tb4_1	E0_1	sector_1	n_1	githash	version	GS_consistency	iterations	dist_function	distance	diff_hybrid	omegaH	script	time	
2L_4b_gen	0.50732753	-1.0095059	0.30156468	4	0.35348769	-0.19849378	0.34381865	-0.19686352	1	1	0.32870347	0.32771589	0.32729181	0.32652732	0.86053263	-1.0095059	nan	nan	nan	nan	nan	0.89232504	-0.60200974	nan	nan	nan	nan	-4.1819256	R0:N6:S0:1	0.89232504	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 3.93e-02	 3.64e-03	 0.50732753	general_bath_1D.py	2023-07-27@12:30
2L_4b_gen	0.51550917	-1.0126196	0.30906244	4	0.35901394	-0.20471213	0.3522231	-0.20262318	1.025	1	0.32390605	0.33055136	0.3227004	0.32965176	0.86476116	-1.0126196	nan	nan	nan	nan	nan	0.89624685	-0.61020643	nan	nan	nan	nan	-4.2414217	R0:N6:S0:1	0.89624685	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 3.93e-02	 1.82e-03	 0.51550917	general_bath_1D.py	2023-07-27@12:45
2L_4b_gen	0.52168737	-1.0043658	0.29597347	4	0.37065675	-0.20965583	0.36637584	-0.20713881	1.05	1	0.32023573	0.33396605	0.31921719	0.33327487	0.87048098	-1.0043658	nan	nan	nan	nan	nan	0.90485242	-0.60936696	nan	nan	nan	nan	-4.2972128	R0:N6:S0:1	0.90485242	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 3.91e-02	 1.81e-03	 0.52168737	general_bath_1D.py	2023-07-27@12:59
2L_4b_gen	0.52396801	-1.0014711	0.31150696	4	0.38512579	-0.21167575	0.3839599	-0.20940951	1.075	1	0.32073433	0.3337681	0.32015489	0.33335797	0.87534773	-1.0014711	nan	nan	nan	nan	nan	0.90977809	-0.61139154	nan	nan	nan	nan	-4.3453144	R0:N6:S0:1	0.90977809	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 3.77e-02	 5.49e-03	 0.52396801	general_bath_1D.py	2023-07-27@13:13
2L_4b_gen	0.5307117	-1.0024757	0.31564497	4	0.39820744	-0.21730264	0.39846486	-0.2151677	1.1	1	0.31643213	0.33663624	0.3160398	0.33635703	0.87986417	-1.0024757	nan	nan	nan	nan	nan	0.91628122	-0.61976507	nan	nan	nan	nan	-4.4032062	R0:N6:S0:1	0.91628122	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 3.81e-02	 1.77e-03	 0.53071170	general_bath_1D.py	2023-07-27@13:28
2L_4b_gen	0.54843193	-0.99168629	0.32034348	4	0.45236559	-0.23041788	0.45642541	-0.22941376	1.15	1	0.31232627	0.34133473	0.31258789	0.34144298	0.88983347	-0.99168629	nan	nan	nan	nan	nan	0.93205628	-0.62790659	nan	nan	nan	nan	-4.5273492	R0:N6:S0:1	0.93205628	20a4d9d	v2.1.5	Y	18	sharp_wc_2.0_b_25	 3.55e-02	 3.21e-03	 0.54843193	general_bath_1D.py	2023-07-27@13:45
2L_4b_gen	0.6251031	-0.97611469	0.33948429	4	0.68012762	-0.27483155	0.68308752	-0.27578679	1.25	1	0.32020348	0.35377712	0.32062771	0.35409161	0.91407045	-0.97611469	nan	nan	nan	nan	nan	0.96308753	-0.65236789	nan	nan	nan	nan	-4.8617141	R0:N6:S0:1	0.96308753	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 2.82e-02	 1.91e-03	 0.62510310	general_bath_1D.py	2023-07-27@14:00
2L_4b_gen	0.7853242	-0.97010015	0.37128631	4	0.99382537	-0.35449102	0.9930616	-0.3545415	1.35	1	0.34187657	0.37516102	0.34183705	0.37513801	0.95167438	-0.97010015	nan	nan	nan	nan	nan	0.98501253	-0.66768056	nan	nan	nan	nan	-5.3686173	R0:N6:S0:1	0.98501253	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 1.62e-02	 1.10e-03	 0.78532420	general_bath_1D.py	2023-07-27@14:14
2L_4b_gen	1.0153854	-0.96114182	0.3893916	4	1.252245	-0.46586091	1.2522822	-0.46585727	1.45	1	0.36972554	0.39273752	0.36972044	0.39274246	0.99809829	-0.96114182	nan	nan	nan	nan	nan	0.9943367	-0.66873521	nan	nan	nan	nan	-6.0254687	R0:N6:S0:1	0.9943367	20a4d9d	v2.1.5	Y	21	sharp_wc_2.0_b_25	 6.83e-03	 4.49e-03	 1.0153854	general_bath_1D.py	2023-07-27@14:33
2L_4b_gen	1.4342945	-0.96120876	0.34273496	4	1.4573048	-0.66681335	1.4572966	-0.66679495	1.55	1	0.40747195	0.41852153	0.40747035	0.4185149	1.0003794	-0.96120876	nan	nan	nan	nan	nan	0.9992715	-0.67198966	nan	nan	nan	nan	-7.0634092	R0:N6:S0:1	0.9992715	20a4d9d	v2.1.5	Y	17	sharp_wc_2.0_b_25	 1.56e-03	 1.66e-03	 1.4342945	general_bath_1D.py	2023-07-27@14:49
2L_4b_gen	1.7101076	-0.96393797	0.28579576	4	1.436189	-0.80153411	1.4360853	-0.80150641	1.65	1	0.41953372	0.42666515	0.41951636	0.42665416	0.99950361	-0.96393797	nan	nan	nan	nan	nan	0.99891422	-0.67462257	nan	nan	nan	nan	-7.8146577	R0:N6:S0:1	0.99891422	20a4d9d	v2.1.5	Y	19	sharp_wc_2.0_b_25	 6.47e-04	 1.96e-03	 1.7101076	general_bath_1D.py	2023-07-27@15:07
2L_4b_gen	1.9541069	-0.96430187	0.41138819	4	1.367098	-0.92100825	1.3671093	-0.92101868	1.75	1	0.42759764	0.43118562	0.42759629	0.4311848	0.99701133	-0.96430187	nan	nan	nan	nan	nan	0.99705005	-0.67397311	nan	nan	nan	nan	-8.5029324	R0:N6:S0:1	0.99705005	20a4d9d	v2.1.5	Y	17	sharp_wc_2.0_b_25	 2.73e-04	 1.57e-03	 1.9541069	general_bath_1D.py	2023-07-27@15:23
2L_4b_gen	2.1960701	-0.96601005	0.30903163	4	1.3281812	-1.0402053	1.3281733	-1.0402263	1.85	1	0.43538897	0.43604928	0.43538672	0.43605343	0.99901338	-0.96601005	nan	nan	nan	nan	nan	0.9990122	-0.67604192	nan	nan	nan	nan	-9.1869979	R0:N6:S0:1	0.9990122	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 1.64e-04	 2.55e-04	 2.1960701	general_bath_1D.py	2023-07-27@15:39
2L_4b_gen	2.5132562	-0.96758452	0.25884204	4	1.1885536	-1.1979957	1.1885282	-1.1979532	2	1	0.43829753	0.43744991	0.43829322	0.43744124	0.99933932	-0.96758452	nan	nan	nan	nan	nan	0.99927072	-0.67731842	nan	nan	nan	nan	-10.121343	R0:N6:S0:1	0.99927072	20a4d9d	v2.1.5	Y	16	sharp_wc_2.0_b_25	 8.03e-05	 5.51e-04	 2.5132562	general_bath_1D.py	2023-07-27@15:55
